src/_guides/technical-writing/5-improvements-to-openapi-operation-documentation.md

```diff
@@ -0,0 +1,255 @@
+    content:
+      application/json:
+        schema:
+          $ref: '#/components/schemas/ProblemDetails'
+  401:
+    description: "Unauthorized - Authentication failed"
+    content:
+      application/json:
+        schema:
+          $ref: '#/components/schemas/ProblemDetails'
+  404:
+    description: "Not Found - Resource doesn’t exist"
+    content:
+      application/json:
+        schema:
+          $ref: '#/components/schemas/ProblemDetails'
+  500:
+    description: "Internal Server Error - System encountered an unexpected error"
+    content:
+      application/json:
+        schema:
+          $ref: '#/components/schemas/ProblemDetails'
+```
+
+## 4. Organize and Tag Your API Operations
+While most developers will never look at the underlying YAML for your reference documentation, most documentation rendering frameworks will render operations in the order that they are captured. This means that related operations that are scattered across your OpenAPI document will make it more difficult for the developer to understand how to use your API and what capabilities it offers. 
+
+At a minimum, group your operations so that they render together and in a thoughtful manner that helps with understanding. You may also wish to use tags to group related operations together. Tags are simply a grouping mechanism that helps documentation frameworks organize the display of your API operations. 
+
+### Example of Disorganized API Operation Documentation
+
+Documentation that lacks structure, such as grouping related endpoints or models, can be challenging to navigate and understand.
+
+**Example Mistake:**
+```yaml
+paths:
+  /users:
+    get: ...
+  /orders:
+    get: ...
+  /users/{id}:
+    get: ...
+```
+
+### Well-Organized and Tagged API Operations
+
+**Improved Example:**
+```yaml
+tags:
+  - name: User Management
+    description: Manages the registered users in the system
+  - name: Order Management
+    description: Supports order creation and status lookup
+paths:
+  /users:
+    get: ...
+      tags:
+        - User Management
+
+  /users/{id}:
+    get: ...
+      tags:
+        - User Management
+  /orders:
+    get: ...
+      tags:
+        - Order Management
+```
+
+## 5. Improve Consistency in Operation Naming Conventions
+
+Inconsistent naming conventions for operations, parameters, and schema properties can lead to confusion and errors in API integration. Since making changes to these conventions can break your consumers, it is important to take a fresh look at your API operations, parameters, and schema definitions before releasing your API. 
+
+To further improve your API design efforts, involve your technical writers earlier in the API design process. This will provide insights regarding improvements that could be made to improve the developer experience. By shifting left your technical writer involvement in the API lifecycle, you also reduce the cost of change due to late feedback. 
```

Comment:
```suggestion
To further improve your API design efforts, involve your technical writers earlier in the API design process. This will provide insights regarding improvements that could be made to improve the developer experience. By shifting your technical writer involvement earlier in the API lifecycle, you also reduce the cost of change due to late feedback. 
```